src/html.ts

```diff
@@ -67,32 +67,39 @@ function makeMergedBundle(exploreResults: ExploreBundleResult[]): ExploreBundleR
     Object.entries(result.files).forEach(([fileName, size]) => {
       files[`${prefix}/${fileName}`] = size;
     });
+    Object.entries(result.filesCoverage).forEach(([fileName, size]) => {
+      filesCoverage[`${prefix}/${fileName}`] = size;
+    });
   }
 
   return {
     bundleName: '[combined]',
     totalBytes,
     unmappedBytes: 0,
     files,
+    filesCoverage,
   };
 }
 
 interface WebTreeMapNode {
   name: string;
   data: {
     $area: number;
+    coveredSize?: number;
   };
   children?: WebTreeMapNode[];
 }
 
 /**
  * Covert file size map to webtreemap data
  */
-function getWebTreeMapData(files: FileSizeMap): WebTreeMapNode {
+function getWebTreeMapData(data: ExploreBundleResult): WebTreeMapNode {
+  const files: FileSizeMap = data.files;
```

Comment:
FileSizeMap is inferenced from ExploreBundleResult.files

Reply:
It shows as a `Record<string, number>` in intellisense, this is more explicit

Reply:
But it's a string -> number dictionary. Moreover, this code block is not that big
```
const { files, filesCoverage } = data;
```
Maybe `files` is not a good name for the property and it should've been named `fileSizeMap`

Reply:
The explicit typing's name has semantic meaning in addition to just the type. I generally prefer to use inferred types over explicit, but I also prefer to use explicit typings over naming the thing after it's type.

I've removed the typing since I don't feel strongly.